sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -272,7 +272,7 @@ package object dsl {
       def attr: UnresolvedAttribute = analysis.UnresolvedAttribute(s)
     }
     implicit class DslAttr(attr: UnresolvedAttribute) extends ImplicitAttribute {
```

Comment:
shall we add `override` to save a roundtrip?

Reply:
Yes, done! 51d838e